### ISSUE TITLE
[Accessibility] Note Editor Toolbar

### ISSUE THREAD

**Issue by david-allison-1** [ankidroid/Anki-Android#8523]
Each of the toolbar items in the Note Editor toolbar should be 48dp (minimum) to meet accessibility standards.

Also note that the expected width of a toolbar item should be changed from 32 (24 +8) dp to (48 + 8) = 56dp.

https://github.com/ankidroid/Anki-Android/blob/c1697ad37158b47d4a44c200e7428704af61792b/AnkiDroid/src/main/java/com/ichi2/anki/noteeditor/Toolbar.java#L209

**Comment by tanmayChakrawarty** [comment]
@david-allison-1  I will work on this.

**Comment by david-allison-1** [comment]
Thanks!!!

**Comment by tanmayChakrawarty** [comment]
@david-allison-1 Is this good or we have to increase the size of each item in the toolbar?

![photo_2021-04-09_16-45-24](https://user-images.githubusercontent.com/75254780/114172352-2e1e3900-9953-11eb-9321-d6e3d9000f8f.jpg)

**Comment by infinyte7** [comment]
Can you try this deck it contains fields more than two? May be toolbar hide `Cards: Card 1` button at bottom with toolbar size increased.
[cloze.zip](https://github.com/ankidroid/Anki-Android/files/6285866/cloze.zip)

**Comment by tanmayChakrawarty** [comment]
@infinyte7 Yes it is hiding the `Cards: Card 1` button at the bottom

![photo_2021-04-09_19-20-41](https://user-images.githubusercontent.com/75254780/114190372-dc80a900-9968-11eb-861a-2a8d6adf713b.jpg)

**Comment by tanmayChakrawarty** [comment]
@infinyte7 @david-allison-1  Hope so this looks good

![photo_2021-04-09_20-00-23](https://user-images.githubusercontent.com/75254780/114195785-6717d700-996e-11eb-9a0c-ee2706b8f659.jpg)

**Comment by infinyte7** [comment]
It looks good. You can create PR for more review.

**Comment by david-allison-1** [comment]
> @david-allison-1 Is this good or we have to increase the size of each item in the toolbar?
> 
> If it is good then I will create the PR.
> 
> ![photo_2021-04-09_16-45-24](https://user-images.githubusercontent.com/75254780/114172352-2e1e3900-9953-11eb-9321-d6e3d9000f8f.jpg)

The spacing for the "add" button looks off

Does a scrollbar appear if more buttons are added? How do these custom buttons look?

**Comment by tanmayChakrawarty** [comment]
@david-allison-1  Yes, the scrollbar is appearing when more buttons are added.


https://user-images.githubusercontent.com/75254780/114210425-663a7180-997d-11eb-9133-88259dd0145c.mp4

**Comment by david-allison-1** [comment]
Awesome! I still think something's up with the Add button's spacing/padding

**Comment by david-allison-1** [comment]
Either way, happy to see a PR when that's working, looks awesome

**Comment by tanmayChakrawarty** [comment]
> Either way, happy to see a PR when that's working, looks awesome

When I am creating a new pull request my previous commits gets included what should I do can you help me?

**Comment by david-allison-1** [comment]
> > Either way, happy to see a PR when that's working, looks awesome
> 
> When I am creating a new pull request my previous commits gets included what should I do can you help me?

Either:

* Checkout master
* Make a new branch
* `git cherry-pick` the commit that you made

----

* `git rebase -i` and `drop` the previous commits which aren't relevant

**Comment by atharva20-coder** [comment]
Is this issue completed? If not can I give it a try?
